Jint/Runtime/Interpreter/Expressions/JintUnaryExpression.cs

```diff
@@ -53,6 +53,7 @@ public override JsValue GetValue()
 
         protected override object EvaluateInternal()
         {
+            var v = _argument.GetValue();
```

Comment:
This might have broken the test suite as it seems that this could have behavioral change.

Reply:
I found some issues with the UnaryExpression fixes, I will add another commit to this soon that should also avoid this behavioral change.